packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx

```diff
@@ -169,66 +173,58 @@ export class GaugeChartBase extends React.Component<IGaugeChartProps, IGaugeChar
                     textAnchor="start"
                     className={this._classNames.limits}
                     role="img"
-                    aria-label={`Max value: ${maxValue}`}
+                    aria-label={`Max value: ${this._maxValue}`}
                   >
-                    {formatValueWithSIPrefix(maxValue)}
+                    {formatValueWithSIPrefix(this._maxValue)}
                   </text>
                 </>
               )}
-              {arcs.map((arc, index) => (
-                <path
-                  key={index}
-                  d={arc.d}
-                  fill={this._segments[arc.segmentIndex].color}
-                  fillOpacity={
-                    this._legendHighlighted(this._segments[arc.segmentIndex].legend) || this._noLegendHighlighted()
-                      ? 1
-                      : 0.1
-                  }
-                  strokeWidth={this.state.focusedElement === this._segments[arc.segmentIndex].legend ? ARC_PADDING : 0}
-                  className={this._classNames.segment}
-                  {...getAccessibleDataObject(
-                    {
-                      ariaLabel: `${this._segments[arc.segmentIndex].legend}, ${
-                        this._segments[arc.segmentIndex].size
-                      } out of ${maxValue - this._minValue} or ${(
-                        (this._segments[arc.segmentIndex].size / (maxValue - this._minValue)) *
-                        100
-                      ).toFixed()}%`,
-                      ...this._segments[arc.segmentIndex].accessibilityData,
-                    },
-                    'img',
-                    true,
-                  )}
-                  onFocus={e => this._handleFocus(e, this._segments[arc.segmentIndex].legend)}
-                  onBlur={this._handleBlur}
-                  onMouseEnter={e => this._handleMouseOver(e, this._segments[arc.segmentIndex].legend)}
-                  onMouseMove={e => this._handleMouseOver(e, this._segments[arc.segmentIndex].legend)}
-                />
-              ))}
+              {arcs.map((arc, index) => {
+                const segment = this._segments[arc.segmentIndex];
+
+                return (
+                  <path
+                    key={index}
+                    d={arc.d}
+                    fill={segment.color}
+                    fillOpacity={this._legendHighlighted(segment.legend) || this._noLegendHighlighted() ? 1 : 0.1}
+                    strokeWidth={this.state.focusedElement === segment.legend ? ARC_PADDING : 0}
+                    className={this._classNames.segment}
+                    {...getAccessibleDataObject(
+                      {
+                        ariaLabel:
+                          this.props.variant === GaugeChartVariant.SingleSegment
+                            ? `${segment.legend}, ${segment.size} out of ${this._maxValue - this._minValue} or ${(
```

Comment:
Similarly here also, narrate out of if min value = 0

Reply:
fixed